flake.nix

```diff
@@ -191,15 +197,19 @@
         packages = inputs.std.harvest inputs.self [ "plutus" "packages" ];
       }
       {
-        hydraJobs = inputs.std.harvest inputs.self [ "automation" "hydraJobs" ];
-        packages = inputs.std.harvest inputs.self [ "automation" "hydraJobs" ];
+        ciJobs = inputs.std.harvest inputs.self [ "automation" "ciJobs" ];
+        packages = inputs.std.harvest inputs.self [ "automation" "ciJobs" ];
       }
       {
         packages = inputs.std.harvest inputs.self [ "toolchain" "packages" ];
       }
       {
         packages = inputs.std.harvest inputs.self [ "toolchain" "scripts" ];
-      };
+      }
+      (inputs.tullia.fromStd {
```

Comment:
Please write down whatever you found out about the infinite recursion. That's useful info, otherwise we'll inevitably try and merge this back in at some point in the future and be very confused!